kumascript/macros/DeprecatedGeneric.ejs

```diff
@@ -20,16 +20,16 @@ if ($1) {
     var t = $1.toLowerCase();
     var n = "";
 
-    if ( string.startswith(t, "js") ) {
-        n = string.trim( string.substr(t, 2) );
+    if (string.startswith(t, "js")) {
+        n = string.trim( string.substr(t, 2));
         ver = "JavaScript ";
         if (string.length(n) > 0) {
-            if(wiki.pageExists(lang + '/docs/JavaScript/New_in_JavaScript/' + n)) {
+            if(true) {
                 link = '/' + lang + '/docs/JavaScript/New_in_JavaScript/' + n;
-            } else if(wiki.pageExists('/en-US/docs/JavaScript/New_in_JavaScript/' + n)) {
+            } else if(true) {
                 link = '/en-US/docs/JavaScript/New_in_JavaScript/' + n;
             }
-        }
+        } 
```

Comment:
Seems your editor/IDE doesn't trim trailing whitespace. I don't think we check for that in `.ejs` files but you should look into setting that up so hitting Cmd-s (or Ctrl-s)  always automatically (right) trims all lines.

Reply:
Good idea. I'm using VS Code and will set that up today.